api/lib/source/polkadotV0-source.js

```diff
@@ -24,22 +29,61 @@ class polkadotAPI {
     this.db = db
   }
 
+  initialize(config) {
+    this.context = config.context
+    // manually set cache to checking it
+    this.cache = new InMemoryLRUCache()
+    this.httpCache = new HTTPCache(this.cache, this.httpFetch)
+  }
+
   setReducers() {
     this.reducers = require('../reducers/polkadotV0-reducers')
   }
 
+  async getRetry(url, intent = 0) {
```

Comment:
refactor and put into a common lib as used by cosmos and polkadot

Reply:
I had problems using apollo RESTdatasource functions (this.get, this.cache, this.memoized) in a external function. I tried to move cache flush logic and clear memoized results to `query()` function and pass this.get func as parameter but I got the error `Cannot read property 'fetch'  of undefined`. Any ideas?